app/helpers/application_helper.rb

```diff
@@ -48,4 +48,13 @@ def body_classes
     classes << "off-canvas" unless @hide_menu
     classes << @shopfront_layout
   end
+
+  def pdf_stylesheet_pack_tag(source)
+    if running_in_development?
```

Comment:
Will this also cover `RAILS_ENV=test`?

Reply:
Yes, I was thinking that too. But how can we test it to be sure? 

Probably @Matt-Yorkley will have a opinion on that 🙏

Reply:
On my local system, pdf invoices are also missing css when running tests (see pic on [this comment](https://github.com/openfoodfoundation/openfoodnetwork/pull/8434#issuecomment-1028965506)). I'm not sure this is a valid test, as there might be other reasons for this...

Reply:
We probably need to open on new issue (based/blocked by this current PR) since the solution might be slightly different.